src/components/ButtonPanel.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import Button from './Button'
+
+const grey = 'rgb(224, 224, 224)'
+
+function ButtonPanel (props) {
```

Comment:
We **can** and we **should** deconstruct `props` to avoid repeating it:
Either this way:
```
function ButtonPanel(props){
  const { clickHandler } = props;
}
```

OR (better way)

```javascript
function({clickHandler}){
 //
}
```
Also we should prefer to use arrow functions over `function` keyword.